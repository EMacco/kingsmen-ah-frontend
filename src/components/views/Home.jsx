import React from 'react';

export default function Home() {
  return <div className='bg-gray-100 font-sans w-full min-h-screen m-0' />;
}
