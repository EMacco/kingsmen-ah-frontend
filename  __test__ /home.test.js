import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Home from '@components/views/Home';

configure({ adapter: new Adapter() });

const setup = () => {
  const wrapper = shallow(<Home />);
  return wrapper;
};

describe('<Home />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setup();
  });

  it('should render the Home component correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
