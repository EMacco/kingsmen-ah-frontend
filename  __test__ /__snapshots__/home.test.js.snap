// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the Home component correctly 1`] = `
<div
  className="bg-gray-100 font-sans w-full min-h-screen m-0"
/>
`;
